src/NServiceBus.AwsLambda.SQS/TransportMessage.cs

```diff
@@ -14,7 +13,7 @@ public TransportMessage()
         {
         }
 
-        public TransportMessage(OutgoingMessage outgoingMessage, List<DeliveryConstraint> deliveryConstraints)
+        public TransportMessage(OutgoingMessage outgoingMessage, List<DispatchProperties> deliveryConstraints)
```

Comment:
doesn't need to be a list I think (see https://github.com/Particular/NServiceBus.AmazonSQS/blob/master/src/NServiceBus.Transport.SQS/TransportMessage.cs)

Reply:
The whole empty message body and body preparation process probably needs an overhaul. We have done quite a bit of changes to make native integration possible. That being said, we can likely do this as subsequent PRs